src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -112,3 +112,132 @@ export default function HomePage(initialData) {
 }
 ```
 
+### Example:
+
+Using the following schema:
+
+```graphql
+type Post
+  @model
+  @auth(rules: [{ allow: owner }, { allow: public, operations: [read] }]) {
+  id: ID!
+  title: String!
+  content: String!
+}
+```
+
+Open **pages/index.js** and replace it with the following code:
+
+```jsx
+// pages/index.js
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import {
+  Amplify,
+  Auth,
+  AuthModeStrategyType,
+  DataStore,
+  withSSRContext,
+} from "aws-amplify";
+import { Authenticator } from "@aws-amplify/ui-react";
+import { Post } from "../src/models";
+import { serializeModel } from "@aws-amplify/datastore/ssr";
+import awsExports from "../src/aws-exports";
+
+Amplify.configure({
+  ...awsExports,
+  DataStore: {
+    authModeStrategyType: AuthModeStrategyType.MULTI_AUTH,
+  },
+  ssr: true,
+});
+
+export async function getServerSideProps({ req }) {
+  const SSR = withSSRContext({ req });
+  const posts = await SSR.DataStore.query(Post);
+
+  return {
+    props: {
+      // 👇 This converts Post instances into serialized JSON for the client
+      posts: serializeModel(posts),
+    },
+  };
+}
+
+async function handleCreatePost(event) {
+  event.preventDefault();
+
+  const form = new FormData(event.target);
+
+  try {
+    const post = await DataStore.save(
+      new Post({
+        title: form.get("title"),
+        content: form.get("content"),
+      })
+    );
+
+    window.location.href = `/posts/${post.id}`;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+export default function Home({ posts = [] }) {
+  console.log(posts);
```

Comment:
Should we have a console.log in this code example?

Reply:
Nope - will remove.